modules/core/src/main/resources/default.scala-steward.conf

```
@@ -31,6 +31,9 @@ updates.ignore = [
   // Artifacts below are ignored because they are not yet announced.
 
   // Ignore the next Scala 3 version until it is announced.
+  { groupId = "org.scala-lang", artifactId = "scala3-compiler",     version = { exact = "3.4.0" } },
+  { groupId = "org.scala-lang", artifactId = "scala3-library",      version = { exact = "3.4.0" } },
+  { groupId = "org.scala-lang", artifactId = "scala3-library_sjs1", version = { exact = "3.4.0" } },
   { groupId = "org.scala-lang", artifactId = "scala3-compiler",     version = { exact = "3.3.2" } },
   { groupId = "org.scala-lang", artifactId = "scala3-library",      version = { exact = "3.3.2" } },
   { groupId = "org.scala-lang", artifactId = "scala3-library_sjs1", version = { exact = "3.3.2" } },
```

I'm not sure if there's a more concise way to express these lines? eg 
```suggestion
  { groupId = "org.scala-lang", artifactId = "scala3-compiler",     version = [ { exact = "3.4.0" }, { exact = "3.3.2" } ]},
  { groupId = "org.scala-lang", artifactId = "scala3-library",      version = [ { exact = "3.4.0" }, { exact = "3.3.2" } ]},
  { groupId = "org.scala-lang", artifactId = "scala3-library_sjs1", version = [ { exact = "3.4.0" }, { exact = "3.3.2" } ]},
```